assets/javascripts/discourse/components/modal/post-event-invitees.js

```diff
@@ -0,0 +1,63 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { debounce } from "discourse-common/utils/decorators";
+import I18n from "I18n";
```

Comment:
One more change that happened when you were away 😄 

```suggestion
import I18n from "discourse-i18n";
```